browser/sync_element_handle_mapping.go

```diff
@@ -90,7 +90,16 @@ func syncMapElementHandle(vu moduleVU, eh *common.ElementHandle) mapping { //nol
 				return nil, eh.Tap(popts) //nolint:wrapcheck
 			}), nil
 		},
-		"textContent":         eh.TextContent,
+		"textContent": func() (any, error) {
```

Comment:
We need to return `T, error` at most to Goja. So, we need to handle these as well.